CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## Changelog (master)
 
+### v8.1.0
+
+#### Features
+* Add preset `babel` as transpilation alternative to `ts-jest` ([#317](https://github.com/thymikee/jest-preset-angular/pull/317)).
```

Comment:
```suggestion
* Expose a new preset `jest-preset-angular/build/babel` for transforming files with Babel instead of `ts-jest` ([#317](https://github.com/thymikee/jest-preset-angular/pull/317)).
```
Side note: would be nice if we could get rid of `build` part, just `jest-preset-angular/babel` – easy enough with re-exporting from top-level.

Reply:
I agree. Maybe we can just publish the `build` folder with the `package.json`? Have to check out how other projects solve this.